pwa/src/pages/endpoints/[endpointId]/handlers/[handlerId].tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import Layout from "../../../../components/common/layout";
+import HandlerForm from "../../../../components/handlers/handlerForm";
+
+const IndexPage = (props) => {
+  const handlerId: string = props.params.handlerId === "new" ? null : props.params.handlerId
+  const endpointId: string = props.params.endpointId === "new" ? null : props.params.endpointId
+
+  return (
+    <Layout title={"Handler properties"} subtext={"Create or edit your properties"}>
+      <main>
+        <div className="row">
+          <div className="col-12">
+            <HandlerForm id={handlerId} endpointId={endpointId} />
+          </div>
+        </div>
+      </main>
+    </Layout>
+  );
+};
+
+export default IndexPage;
```

Comment:
Missing a white line at the end of this file (just add one enter after line `22`).

This has to do with some magic Github does for our code management (see screenshot, Github actually asks us to add it):

![image](https://user-images.githubusercontent.com/66552932/154635158-212da1fe-312b-42cf-a04b-b0dd8d6d8244.png)